client/lib/cart-values/test/index.js

```diff
@@ -3,7 +3,7 @@
 /**
  * External dependencies
  */
-import assert from 'assert';
+import assert from 'assert'; // eslint-disable-line import/no-nodejs-modules
```

Comment:
It would be awesome and quite simple to migrate to `import { assert } from 'chai'` here. But that's for another PR. A codemod would help a lot with the migration for sure.

Reply:
a small find/replace would probably work as well.
I'll leave that for a small follow-up pr